ChangeLog.md

```diff
@@ -13,6 +13,10 @@
   - Continuous integration does not use Travis anymore but Github
     Actions. (David Coeurjolly, [#1591](https://github.com/DGtal-team/DGtal/pull/1591))
 
+- *Geometry package*
+  - Bug fix in ExamplePlaneProbingSurfaceLocalEstimato.
```

Comment:
```suggestion
  - Bug fix in examplePlaneProbingSurfaceLocalEstimator.
```

Reply:
Actually, there are not so much changes. The diff gives a somewhat erroneous impression. The most significant changes are: 

- in `PlaneProbingDigitalSurfaceLocalEstimator`
  - `probingFrameWithPreEstimation` now returns a pair bool-frame instead of just a frame, in order to tell whether the frame will lead to a valid initialization or not. 
  - `eval` now uses this boolean value to return the trivial normal vector if it has been set to 'False' (which means that no valid initialization is possible)
- in `PlaneProbingParallelepipedEstimator`: `isValid` does not call the `isValid` method of the delegate, but only checks the relevant parts (which have been pushed in to separate methods). 

In addition, there are a few cosmetic changes: `InPlane` -> `inPlane` and some function definitions have been put in the .ih files.